galactory/utilities.py

```diff
@@ -51,15 +52,21 @@ def _session_with_retries(retry=None, auth=None) -> Session:
 def authorize(request: Request, artifactory_path: ArtifactoryPath, retry=None, skip_configured_key: bool = False) -> ArtifactoryPath:
     auth = None
     apikey = None
+    bearerToken = None
     if not skip_configured_key:
         apikey = current_app.config['ARTIFACTORY_API_KEY']
 
+    if not skip_configured_key:
+        bearerToken = current_app.config['ARTIFACTORY_BEARER_TOKEN']
```

Comment:
This line can move to 58, so it's under the same conditional as above, then we can have

```python
    if not skip_configured_key:
        apikey = current_app.config['ARTIFACTORY_API_KEY']
        bearerToken = current_app.config['ARTIFACTORY_BEARER_TOKEN']
```